bin/ares-package.js

```diff
@@ -326,6 +296,44 @@ PalmPackage.prototype = {
             this.appOk.bind(this));
```

Comment:
this.finish.bind(this, null, {msg:"no problems detected"}));

Reply:
Done.